### PR TITLE
Revisions and remarks on DAPS user guide

### PR DESCRIPTION
**Pull request by dmpop**
Edited some passages for clarity, added remarks and ideas.

### CODE REVIEW

**Review by tomschr — CHANGES_REQUESTED** [review]
Let's test it.

File: doc/xml/art_daps_quick.xml [hunk 1]
```diff
@@ -111,13 +111,14 @@
...
+    output), refer to its installation instructions (or to your system
     administrator). For FOP, you usually only need to install the respective
     FOP package. However, not all FOP packages contain the hyphenation pattern
-    files. Using the hyphenation patterns is recommended.
-    <remark>taroth 2012-03-26: todo - mention download from
-   offo.sourceforge.net in troubleshooting section and add xref</remark>
+    files. Using the hyphenation patterns is recommended.  <remark>taroth
+    2012-03-26: todo - mention download from offo.sourceforge.net in
+    troubleshooting section and add xref</remark> <remark>dpopov 2016-10-10:
+    What's FOP?</remark>
```
Comment (tomschr) [inline doc/xml/art_daps_quick.xml:121]: This is just a comment. Ignore it. :)

File: doc/xml/art_daps_quick.xml [hunk 2]
```diff
@@ -185,7 +186,7 @@
     <callout arearefs="co.quick.docdir.img.src">
      <para>
       Top-level directory for any original images that you want to use in
-      the documentation project. Contains subdirectories for images in
+      the documentation project. This directory contains subdirectories for images in
```
Comment (tomschr) [inline doc/xml/art_daps_quick.xml:189]: Here is another comment.

**Comment by dmpop** [comment]
OK, let's test it.

**Review by tomschr — APPROVED** [review]
Let's approve

**Pull request merged**